packages/scss/src/components/_actionIcon.scss

```diff
@@ -10,11 +10,24 @@
 	width: 2rem;
 
 	&:hover {
-		background: _component("actionIcon.hover.background");
-		color: darken(_color("text.default"), 25%);
+		&:not([disabled]) {
```

Comment:
Ok, I used the same code that we used for buttons. 
A class is-disabled has been added with a pointer event none.
I also fixed opacity which was not linked to common.disabled.